hack/test-cmd.sh

```diff
@@ -236,6 +236,7 @@ runTests() {
   kube::test::get_object_assert 'pods/valid-pod' "{{$id_field}}" 'valid-pod'
   # Describe command should print detailed information
   kube::test::describe_object_assert pods 'valid-pod' "Name:" "Image(s):" "Node:" "Labels:" "Status:" "Replication Controllers"
+  kubectl describe "${kube_flags[@]}" pods
```

Comment:
This checks that command runs and exits 0, but not output. I sent you a short pr to improve coverage a bit.